packages/amplify_core/lib/src/types/datastore/hub/datstore_hub_event.dart

```diff
@@ -3,13 +3,74 @@
 
 import 'package:amplify_core/amplify_core.dart';
 
+/// {@template datastore_hub_event}
```

Comment:
These template tags must be globally unique across all packages. Always prefix with the package name by habit.

```suggestion
/// {@template amplify_core.datastore_hub_event}
```